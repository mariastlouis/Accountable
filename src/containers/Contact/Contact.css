@import url("https://fonts.googleapis.com/css?family=Pathway+Gothic+One");
@import url("https://fonts.googleapis.com/css?family=Cabin");
@import url("https://fonts.googleapis.com/css?family=Open+Sans:300,400,600");
@import url("https://fonts.googleapis.com/css?family=Raleway:300,400,500,600");
body {
  font-family: "Open Sans", sans-serif;
  color: #545252; }

h1, h2, h3 {
  font-family: "Raleway", sans-serif;
  -webkit-margin-before: 0;
  -webkit-margin-after: 0;
  -webkit-margin-start: 0px;
  -webkit-margin-end: 0px; }

h1 {
  font-size: 40px;
  font-weight: 400; }

h2 {
  font-weight: 300;
  display: block; }

p {
  font-size: 16px;
  font-family: "Open Sans", sans-serif;
  font-weight: 400; }

hr {
  border: 0;
  height: 1px;
  background: #333; }

.Contact {
  margin: 20px; }

h1.accent-hed {
  background: #3a6985;
  color: white;
  display: flex;
  justify-content: center;
  font-size: 36px;
  padding: 8px;
  max-width: 1200px;
  margin-left: auto;
  margin-right: auto; }

.main-contact-content {
  display: flex;
  justify-content: center; }

.lawmaker-profile {
  max-width: 1000px;
  display: flex;
  align-items: center;
  flex-direction: row;
  justify-content: space-between;
  margin-top: 20px;
  margin-left: auto;
  margin-right: auto; }

.mugshot-image {
  width: 150px;
  height: 150px;
  position: relative;
  overflow: hidden;
  -webkit-border-radius: 50%;
  -moz-border-radius: 50%;
  -ms-border-radius: 50%;
  -o-border-radius: 50%;
  border-radius: 50%; }

.mugshot {
  height: 250px; }

ul {
  list-style-type: none; }

li {
  line-height: 20px; }

p.right-side-info {
  display: block;
  -webkit-margin-before: 1em;
  -webkit-margin-after: 1em;
  -webkit-margin-start: 0px;
  -webkit-margin-end: 0px; }

.label {
  text-transform: uppercase;
  font-weight: 600; }

.secondary-hed {
  font-size: 30px;
  font-weight: 600; }

.bill-title {
  font-weight: 600; }

.committees {
  margin-left: 20px;
  margin-right: 20px; }

@media screen and (max-width: 600px) {
  .lawmaker-profile {
    flex-direction: column;
    align-items: center; } }
