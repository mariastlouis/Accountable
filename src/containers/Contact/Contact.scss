@import '../../assets/variable.scss';

.Contact {
  margin: 20px;
}


h1.chamber-hed {
    background: $accent-blue;
    color: white;
    display: flex;
    justify-content: center;
    font-size: 36px;
    padding: 8px;
}


.main-contact-content {
  display: flex;
  flex-direction: row;

}

.left-side-info {
  width: 50%;
}

.lawmaker-name{
  text-align: center;
}

.mugshot-image {
  width: 150px;
  height: 150px;
  position: relative;
  overflow: hidden;
  -webkit-border-radius: 50%;
  -moz-border-radius: 50%;
  -ms-border-radius: 50%;
  -o-border-radius: 50%;
  border-radius: 50%;
}
.mugshot {
  display: inline;
  margin: 0 auto;
  width: auto;
  height: 250px;
}

ul {
  list-style-type: none
}

li {
  line-height: 20px;
}

.right-side-info {
  margin-top: 30px;
}
