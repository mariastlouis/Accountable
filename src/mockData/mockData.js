export const mockLawmakerApiResponse = [
    {
        "last_name": "Duran",
        "updated_at": "2017-12-20 07:08:21",
        "full_name": "Crisanta Duran",
        "id": "COL000067",
        "first_name": "Crisanta",
        "middle_name": "",
        "district": "5",
        "chamber": "lower",
        "state": "co",
        "votesmart_id": "125477",
        "party": "Democratic",
        "suffixes": "",
        "email": "",
        "all_ids": [
            "COL000067"
        ],
        "leg_id": "COL000067",
        "active": true,
        "transparencydata_id": "75bb4e6db9f946caa09e9d84256b85a8",
        "photo_url": "http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_h-speaker%20duran%20-17.jpg?itok=573JhyU2",
        "level": "state",
        "url": "http://leg.colorado.gov/legislators/crisanta-duran",
        "country": "us",
        "created_at": "2012-01-19 01:01:49",
        "+occupation": "Attorney",
        "offices": [
            {
                "fax": null,
                "name": "Capitol Office",
                "phone": null,
                "address": "200 E. Colfax\nRM 307\nDenver, CO 80203",
                "type": "capitol",
                "email": null
            }
        ],
        "+phone": "303 866 2925"
    }
]

export const mockLawmakerObject = {
    id: "COL000067",
    contact: {
        firstName: "Crisanta",
        lastName: "Duran",
        party: "Democratic",
        image: "http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_h-speaker%20duran%20-17.jpg?itok=573JhyU2",
        website: "http://leg.colorado.gov/legislators/crisanta-duran",
        phone: "303 866 2925",
        active: true,
        occuption: "Attorney",
        chamber: "lower",
        district:"5"
    }
}

export const mockDetailApiCall {
  "last_name": "Duran",
  "updated_at": "2017-12-22 07:08:43",
  "sources": [
    {
      "url": "http://leg.colorado.gov/legislators/crisanta-duran"
    },
    {
      "url": "http://leg.colorado.gov/legislators"
    }
  ],
  "full_name": "Crisanta Duran",
  "old_roles": {
    "2013-2014": [
      {
        "term": "2013-2014",
        "end_date": null,
        "district": "5",
        "chamber": "lower",
        "state": "co",
        "party": "Democratic",
        "type": "member",
        "start_date": null
      }
    ],
    "2011-2012": [
      {
        "term": "2011-2012",
        "end_date": null,
        "district": "5",
        "chamber": "lower",
        "state": "co",
        "party": "Democratic",
        "type": "member",
        "start_date": null
      },
      {
        "term": "2011-2012",
        "committee_id": "COC000009",
        "end_date": null,
        "chamber": "lower",
        "state": "co",
        "committee": "Judiciary",
        "position": "member",
        "type": "committee member",
        "start_date": null
      },
      {
        "term": "2011-2012",
        "committee_id": "COC000022",
        "end_date": null,
        "chamber": "lower",
        "state": "co",
        "committee": "State, Veterans, & Military Affairs",
        "position": "member",
        "type": "committee member",
        "start_date": null
      },
      {
        "term": "2011-2012",
        "committee_id": "COC000015",
        "end_date": null,
        "chamber": "lower",
        "state": "co",
        "committee": "Legislative Council",
        "position": "member",
        "type": "committee member",
        "start_date": null
      }
    ],
    "2015-2016": [
      {
        "term": "2015-2016",
        "end_date": null,
        "district": "5",
        "chamber": "lower",
        "state": "co",
        "party": "Democratic",
        "type": "member",
        "start_date": null
      },
      {
        "term": "2015-2016",
        "committee_id": "COC000122",
        "chamber": "joint",
        "state": "co",
        "subcommittee": null,
        "committee": "Legislative Council",
        "position": "member",
        "type": "committee member"
      },
      {
        "term": "2015-2016",
        "committee_id": "COC000147",
        "chamber": "joint",
        "state": "co",
        "subcommittee": null,
        "committee": "Executive Committee of the Legislative Council",
        "position": "member",
        "type": "committee member"
      }
    ]
  },
  "id": "COL000067",
  "first_name": "Crisanta",
  "middle_name": "",
  "district": "5",
  "chamber": "lower",
  "state": "co",
  "votesmart_id": "125477",
  "party": "Democratic",
  "suffixes": "",
  "email": "",
  "all_ids": [
    "COL000067"
  ],
  "leg_id": "COL000067",
  "active": true,
  "transparencydata_id": "75bb4e6db9f946caa09e9d84256b85a8",
  "photo_url": "http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_h-speaker%20duran%20-17.jpg?itok=573JhyU2",
  "roles": [
    {
      "term": "2017-2018",
      "end_date": null,
      "district": "5",
      "chamber": "lower",
      "state": "co",
      "party": "Democratic",
      "type": "member",
      "start_date": null
    }
  ],
  "level": "state",
  "url": "http://leg.colorado.gov/legislators/crisanta-duran",
  "country": "us",
  "created_at": "2012-01-19 01:01:49",
  "+occupation": "Attorney",
  "offices": [
    {
      "fax": null,
      "name": "Capitol Office",
      "phone": null,
      "address": "200 E. Colfax\nRM 307\nDenver, CO 80203",
      "type": "capitol",
      "email": null
    }
  ],
  "+phone": "303 866 2925"
}
