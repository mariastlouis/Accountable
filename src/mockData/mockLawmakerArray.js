export const mockLawmakerArray = [
{
  id: 'COL000043',
  isSelected: false,
  contact: {
    firstName: 'Jon',
    lastName: 'Becker',
    party: 'Republican',
    image: "http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_rsz_beckerjon-co-17.jpg?itok=MZ7fny_6",
    website: "http://leg.colorado.gov/legislators/jon-becker",
    phone: "303 866 2906",
    active: true,
    occupation: "Telecom Executive",
    chamber: 'lower',
    district: '65'
    }
  },
  {
  id: 'COL000067',
  isSelected: false,
  contact: {
    firstName: "Crisanta",
    lastName: 'Duran',
    party: 'Democratic',
    image: "http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_h-speaker%20duran%20-17.jpg?itok=573JhyU2",
    website: "http://leg.colorado.gov/legislators/crisanta-duran",
    phone: "303 866 2925",
    active: true,
    occupation: "Attorney",
    chamber: 'lower',
    district: '5'
  }
}
]
