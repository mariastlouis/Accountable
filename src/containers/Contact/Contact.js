import React from 'react';
import PropTypes from 'prop-types';
import './Contact.css'


const Contact = ({id, firstName, lastName, party, image, website, phone, occupation, district, chamber, committees}) => {
  
  const label = (field, label) => {
    return field ? label : null;
  }
  return (

    <div className = "Contact">
      <div className = "left-side-info">
        <div className = "lawmaker-name">
          <h2> {firstName} {lastName} </h2>
        </div>
        <div className = "intro">
          <div className = "mugshot-image">
            <img className = "mugshot" src = {image} alt = {`${firstName} ${lastName}`} />
          </div>
          <ul>

          <li> {party} </li>
         
          <li> {chamber} District {district} </li>
          </ul>
        </div>
      </div>
      <div className = "right-side-info">
        <ul>
          <li> <a href = {website}> {firstName} {lastName}'s Website </a> </li>
          <li> {label(phone, 'Phone:')} {phone} </li>
          <li> {label(occupation, 'Occuption:')} {occupation} </li>
          <li> {id} </li>

        </ul>
      </div>
    </div>
  );
};

export default Contact;