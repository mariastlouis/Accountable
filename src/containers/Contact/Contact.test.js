import  Contact  from './Contact.js';
import React from 'react';
import { shallow } from 'enzyme';
import { mockSelectedLawmaker } from '../../mockData/mockSelectedLawmaker';

describe('Contact component tests', () => {
  let mockLawmakerDetails = mockSelectedLawmaker;
  let renderedComponent;

  beforeEach(() => {
    renderedComponent = shallow(
      <Contact 
        id = {mockLawmakerDetails.id}
        firstName = {mockLawmakerDetails.contact.firstName}
        lastName = {mockLawmakerDetails.contact.lastName}
        party= {mockLawmakerDetails.contact.party}
        image = {mockLawmakerDetails.contact.image}
        website = {mockLawmakerDetails.contact.website}
        phone = {mockLawmakerDetails.contact.phone}
        occupation = {mockLawmakerDetails.contact.occupation}
        district = {mockLawmakerDetails.contact.district}
        chamber = {mockLawmakerDetails.contact.chamber}
      />);

  });

  it('should render', () =>{
    expect(renderedComponent).toBeDefined();
  });

  it('should have a class of Contact, an accent head and a secondary head', () => {
    expect(renderedComponent.find('.Contact').length).toEqual(1);
    expect(renderedComponent.find('.accent-hed').length).toEqual(1);
    expect(renderedComponent.find('.secondary-hed').length).toEqual(1);
  });

  it('should match the snapshot', () => {
    expect(renderedComponent).toMatchSnapshot();
  });

});