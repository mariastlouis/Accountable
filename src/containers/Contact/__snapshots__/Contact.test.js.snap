// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact component tests should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Contact
    chamber="Senate"
    district={34}
    firstName="Lucia"
    id="COL000033"
    image="http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_rsz_guzman-co-17.jpg?itok=kF6sK0P4"
    lastName="Guzman"
    occupation="Senator, District 34"
    party="Democrat"
    phone="303 866 4862"
    website="http://leg.colorado.gov/legislators/lucia-guzman"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1
          className="accent-hed"
>
           
          Senate
           District 
          34
</h1>,
        <div
          className="main-contact-content"
>
          <div
                    className="left-side-info"
          >
                    <img
                              alt="Lucia Guzman"
                              className="mugshot"
                              src="http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_rsz_guzman-co-17.jpg?itok=kF6sK0P4"
                    />
          </div>
          <div
                    className="right-side-info"
          >
                    <div
                              className="lawmaker-name"
                    >
                              <h2
                                        className="secondary-hed"
                              >
                                         
                                        Lucia
                                         
                                        Guzman
                                         
                              </h2>
                    </div>
                    <p>
                               
                              <span
                                        className="label"
                              >
                                        Party:
                              </span>
                               
                              Democrat
                    </p>
                    <p>
                               
                              <span
                                        className="label"
                              >
                                         
                                        Website:
                                         
                              </span>
                              <a
                                        href="http://leg.colorado.gov/legislators/lucia-guzman"
                              >
                                         
                                        Lucia
                                         
                                        Guzman
                                        's Website 
                              </a>
                               
                    </p>
                    <p>
                               
                              <span
                                        className="label"
                              >
                                        Phone:
                                         
                              </span>
                               
                              303 866 4862
                               
                    </p>
                    <p>
                               
                              <span
                                        className="label"
                              >
                                        Occupation:
                                         
                              </span>
                               
                              Senator, District 34
                               
                    </p>
          </div>
</div>,
        <hr />,
      ],
      "className": "Contact",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            " ",
            "Senate",
            " District ",
            34,
          ],
          "className": "accent-hed",
        },
        "ref": null,
        "rendered": Array [
          " ",
          "Senate",
          " District ",
          34,
        ],
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="left-side-info"
>
              <img
                            alt="Lucia Guzman"
                            className="mugshot"
                            src="http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_rsz_guzman-co-17.jpg?itok=kF6sK0P4"
              />
</div>,
            <div
              className="right-side-info"
>
              <div
                            className="lawmaker-name"
              >
                            <h2
                                          className="secondary-hed"
                            >
                                           
                                          Lucia
                                           
                                          Guzman
                                           
                            </h2>
              </div>
              <p>
                             
                            <span
                                          className="label"
                            >
                                          Party:
                            </span>
                             
                            Democrat
              </p>
              <p>
                             
                            <span
                                          className="label"
                            >
                                           
                                          Website:
                                           
                            </span>
                            <a
                                          href="http://leg.colorado.gov/legislators/lucia-guzman"
                            >
                                           
                                          Lucia
                                           
                                          Guzman
                                          's Website 
                            </a>
                             
              </p>
              <p>
                             
                            <span
                                          className="label"
                            >
                                          Phone:
                                           
                            </span>
                             
                            303 866 4862
                             
              </p>
              <p>
                             
                            <span
                                          className="label"
                            >
                                          Occupation:
                                           
                            </span>
                             
                            Senator, District 34
                             
              </p>
</div>,
          ],
          "className": "main-contact-content",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <img
                alt="Lucia Guzman"
                className="mugshot"
                src="http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_rsz_guzman-co-17.jpg?itok=kF6sK0P4"
/>,
              "className": "left-side-info",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "Lucia Guzman",
                "className": "mugshot",
                "src": "http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_rsz_guzman-co-17.jpg?itok=kF6sK0P4",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="lawmaker-name"
>
                  <h2
                                    className="secondary-hed"
                  >
                                     
                                    Lucia
                                     
                                    Guzman
                                     
                  </h2>
</div>,
                <p>
                   
                  <span
                                    className="label"
                  >
                                    Party:
                  </span>
                   
                  Democrat
</p>,
                <p>
                   
                  <span
                                    className="label"
                  >
                                     
                                    Website:
                                     
                  </span>
                  <a
                                    href="http://leg.colorado.gov/legislators/lucia-guzman"
                  >
                                     
                                    Lucia
                                     
                                    Guzman
                                    's Website 
                  </a>
                   
</p>,
                <p>
                   
                  <span
                                    className="label"
                  >
                                    Phone:
                                     
                  </span>
                   
                  303 866 4862
                   
</p>,
                <p>
                   
                  <span
                                    className="label"
                  >
                                    Occupation:
                                     
                  </span>
                   
                  Senator, District 34
                   
</p>,
              ],
              "className": "right-side-info",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <h2
                    className="secondary-hed"
>
                     
                    Lucia
                     
                    Guzman
                     
</h2>,
                  "className": "lawmaker-name",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      " ",
                      "Lucia",
                      " ",
                      "Guzman",
                      " ",
                    ],
                    "className": "secondary-hed",
                  },
                  "ref": null,
                  "rendered": Array [
                    " ",
                    "Lucia",
                    " ",
                    "Guzman",
                    " ",
                  ],
                  "type": "h2",
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    " ",
                    <span
                      className="label"
>
                      Party:
</span>,
                    " ",
                    "Democrat",
                  ],
                },
                "ref": null,
                "rendered": Array [
                  " ",
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Party:",
                      "className": "label",
                    },
                    "ref": null,
                    "rendered": "Party:",
                    "type": "span",
                  },
                  " ",
                  "Democrat",
                ],
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    " ",
                    <span
                      className="label"
>
                       
                      Website:
                       
</span>,
                    <a
                      href="http://leg.colorado.gov/legislators/lucia-guzman"
>
                       
                      Lucia
                       
                      Guzman
                      's Website 
</a>,
                    " ",
                  ],
                },
                "ref": null,
                "rendered": Array [
                  " ",
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        " ",
                        "Website:",
                        " ",
                      ],
                      "className": "label",
                    },
                    "ref": null,
                    "rendered": Array [
                      " ",
                      "Website:",
                      " ",
                    ],
                    "type": "span",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        " ",
                        "Lucia",
                        " ",
                        "Guzman",
                        "'s Website ",
                      ],
                      "href": "http://leg.colorado.gov/legislators/lucia-guzman",
                    },
                    "ref": null,
                    "rendered": Array [
                      " ",
                      "Lucia",
                      " ",
                      "Guzman",
                      "'s Website ",
                    ],
                    "type": "a",
                  },
                  " ",
                ],
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    " ",
                    <span
                      className="label"
>
                      Phone:
                       
</span>,
                    " ",
                    "303 866 4862",
                    " ",
                  ],
                },
                "ref": null,
                "rendered": Array [
                  " ",
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        "Phone:",
                        " ",
                      ],
                      "className": "label",
                    },
                    "ref": null,
                    "rendered": Array [
                      "Phone:",
                      " ",
                    ],
                    "type": "span",
                  },
                  " ",
                  "303 866 4862",
                  " ",
                ],
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    " ",
                    <span
                      className="label"
>
                      Occupation:
                       
</span>,
                    " ",
                    "Senator, District 34",
                    " ",
                  ],
                },
                "ref": null,
                "rendered": Array [
                  " ",
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        "Occupation:",
                        " ",
                      ],
                      "className": "label",
                    },
                    "ref": null,
                    "rendered": Array [
                      "Occupation:",
                      " ",
                    ],
                    "type": "span",
                  },
                  " ",
                  "Senator, District 34",
                  " ",
                ],
                "type": "p",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "hr",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1
            className="accent-hed"
>
             
            Senate
             District 
            34
</h1>,
          <div
            className="main-contact-content"
>
            <div
                        className="left-side-info"
            >
                        <img
                                    alt="Lucia Guzman"
                                    className="mugshot"
                                    src="http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_rsz_guzman-co-17.jpg?itok=kF6sK0P4"
                        />
            </div>
            <div
                        className="right-side-info"
            >
                        <div
                                    className="lawmaker-name"
                        >
                                    <h2
                                                className="secondary-hed"
                                    >
                                                 
                                                Lucia
                                                 
                                                Guzman
                                                 
                                    </h2>
                        </div>
                        <p>
                                     
                                    <span
                                                className="label"
                                    >
                                                Party:
                                    </span>
                                     
                                    Democrat
                        </p>
                        <p>
                                     
                                    <span
                                                className="label"
                                    >
                                                 
                                                Website:
                                                 
                                    </span>
                                    <a
                                                href="http://leg.colorado.gov/legislators/lucia-guzman"
                                    >
                                                 
                                                Lucia
                                                 
                                                Guzman
                                                's Website 
                                    </a>
                                     
                        </p>
                        <p>
                                     
                                    <span
                                                className="label"
                                    >
                                                Phone:
                                                 
                                    </span>
                                     
                                    303 866 4862
                                     
                        </p>
                        <p>
                                     
                                    <span
                                                className="label"
                                    >
                                                Occupation:
                                                 
                                    </span>
                                     
                                    Senator, District 34
                                     
                        </p>
            </div>
</div>,
          <hr />,
        ],
        "className": "Contact",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              " ",
              "Senate",
              " District ",
              34,
            ],
            "className": "accent-hed",
          },
          "ref": null,
          "rendered": Array [
            " ",
            "Senate",
            " District ",
            34,
          ],
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="left-side-info"
>
                <img
                                alt="Lucia Guzman"
                                className="mugshot"
                                src="http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_rsz_guzman-co-17.jpg?itok=kF6sK0P4"
                />
</div>,
              <div
                className="right-side-info"
>
                <div
                                className="lawmaker-name"
                >
                                <h2
                                                className="secondary-hed"
                                >
                                                 
                                                Lucia
                                                 
                                                Guzman
                                                 
                                </h2>
                </div>
                <p>
                                 
                                <span
                                                className="label"
                                >
                                                Party:
                                </span>
                                 
                                Democrat
                </p>
                <p>
                                 
                                <span
                                                className="label"
                                >
                                                 
                                                Website:
                                                 
                                </span>
                                <a
                                                href="http://leg.colorado.gov/legislators/lucia-guzman"
                                >
                                                 
                                                Lucia
                                                 
                                                Guzman
                                                's Website 
                                </a>
                                 
                </p>
                <p>
                                 
                                <span
                                                className="label"
                                >
                                                Phone:
                                                 
                                </span>
                                 
                                303 866 4862
                                 
                </p>
                <p>
                                 
                                <span
                                                className="label"
                                >
                                                Occupation:
                                                 
                                </span>
                                 
                                Senator, District 34
                                 
                </p>
</div>,
            ],
            "className": "main-contact-content",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <img
                  alt="Lucia Guzman"
                  className="mugshot"
                  src="http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_rsz_guzman-co-17.jpg?itok=kF6sK0P4"
/>,
                "className": "left-side-info",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "Lucia Guzman",
                  "className": "mugshot",
                  "src": "http://leg.colorado.gov/sites/default/files/styles/width_300/public/2017b_rsz_guzman-co-17.jpg?itok=kF6sK0P4",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <div
                    className="lawmaker-name"
>
                    <h2
                                        className="secondary-hed"
                    >
                                         
                                        Lucia
                                         
                                        Guzman
                                         
                    </h2>
</div>,
                  <p>
                     
                    <span
                                        className="label"
                    >
                                        Party:
                    </span>
                     
                    Democrat
</p>,
                  <p>
                     
                    <span
                                        className="label"
                    >
                                         
                                        Website:
                                         
                    </span>
                    <a
                                        href="http://leg.colorado.gov/legislators/lucia-guzman"
                    >
                                         
                                        Lucia
                                         
                                        Guzman
                                        's Website 
                    </a>
                     
</p>,
                  <p>
                     
                    <span
                                        className="label"
                    >
                                        Phone:
                                         
                    </span>
                     
                    303 866 4862
                     
</p>,
                  <p>
                     
                    <span
                                        className="label"
                    >
                                        Occupation:
                                         
                    </span>
                     
                    Senator, District 34
                     
</p>,
                ],
                "className": "right-side-info",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <h2
                      className="secondary-hed"
>
                       
                      Lucia
                       
                      Guzman
                       
</h2>,
                    "className": "lawmaker-name",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        " ",
                        "Lucia",
                        " ",
                        "Guzman",
                        " ",
                      ],
                      "className": "secondary-hed",
                    },
                    "ref": null,
                    "rendered": Array [
                      " ",
                      "Lucia",
                      " ",
                      "Guzman",
                      " ",
                    ],
                    "type": "h2",
                  },
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      " ",
                      <span
                        className="label"
>
                        Party:
</span>,
                      " ",
                      "Democrat",
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    " ",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Party:",
                        "className": "label",
                      },
                      "ref": null,
                      "rendered": "Party:",
                      "type": "span",
                    },
                    " ",
                    "Democrat",
                  ],
                  "type": "p",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      " ",
                      <span
                        className="label"
>
                         
                        Website:
                         
</span>,
                      <a
                        href="http://leg.colorado.gov/legislators/lucia-guzman"
>
                         
                        Lucia
                         
                        Guzman
                        's Website 
</a>,
                      " ",
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    " ",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          " ",
                          "Website:",
                          " ",
                        ],
                        "className": "label",
                      },
                      "ref": null,
                      "rendered": Array [
                        " ",
                        "Website:",
                        " ",
                      ],
                      "type": "span",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          " ",
                          "Lucia",
                          " ",
                          "Guzman",
                          "'s Website ",
                        ],
                        "href": "http://leg.colorado.gov/legislators/lucia-guzman",
                      },
                      "ref": null,
                      "rendered": Array [
                        " ",
                        "Lucia",
                        " ",
                        "Guzman",
                        "'s Website ",
                      ],
                      "type": "a",
                    },
                    " ",
                  ],
                  "type": "p",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      " ",
                      <span
                        className="label"
>
                        Phone:
                         
</span>,
                      " ",
                      "303 866 4862",
                      " ",
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    " ",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          "Phone:",
                          " ",
                        ],
                        "className": "label",
                      },
                      "ref": null,
                      "rendered": Array [
                        "Phone:",
                        " ",
                      ],
                      "type": "span",
                    },
                    " ",
                    "303 866 4862",
                    " ",
                  ],
                  "type": "p",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      " ",
                      <span
                        className="label"
>
                        Occupation:
                         
</span>,
                      " ",
                      "Senator, District 34",
                      " ",
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    " ",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          "Occupation:",
                          " ",
                        ],
                        "className": "label",
                      },
                      "ref": null,
                      "rendered": Array [
                        "Occupation:",
                        " ",
                      ],
                      "type": "span",
                    },
                    " ",
                    "Senator, District 34",
                    " ",
                  ],
                  "type": "p",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "hr",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
